### ISSUE TITLE
Post CSD authoring quickstart training material

### ISSUE THREAD

**Issue by ronaldtse** [riboseinc/metanorma.com#25]
Uploaded here now: 
https://github.com/riboseinc/csd-metanorma-training

CC: @strogonoff @opoudjis

**Comment by strogonoff** [comment]
@ronaldtse what exactly was the target audience for this training?

**Comment by ronaldtse** [comment]
Authors of CalConnect Standards. It mostly applies to all standard authors, it’s just that the format “CSD” is specific to CalConnect. 

To adapt for other standard formats (ISO, GB, IETF) all we need to is change “CSD” written in the training to the target standard name.

**Comment by strogonoff** [comment]
Who normally authors those standards? Was there a training event of sorts they attended?

**Comment by ronaldtse** [comment]
Authors are technical experts of standards bodies, such as CalConnect.

This event was: https://www.calconnect.org/events/calconnect-xliii-september-2018

> Wednesday Afternoon 26 September - TECHNICAL TRACK
> 1330-1430 | Authoring CalConnect Standards Lab
> Using Asciidoc and GitHub Repositories to author and publish CalConnect Standards

**Comment by strogonoff** [comment]
Thanks for the link, helps understand the context.

**Comment by strogonoff** [comment]
@ronaldtse just curious, is there a video recording of the presentation?

**Comment by strogonoff** [comment]
It would make sense to publish it in a form of a blog post as well

**Comment by ronaldtse** [comment]
Agreed. Unfortunately there wasn’t a recording, but you should ask @ribose-jeffreylau for a Skype re-run :wink:

**Comment by strogonoff** [comment]
Understood, I’ll work off the slides for now to save the time and we’ll improve on top of that later

**Comment by strogonoff** [comment]
Quickstart & sample doc added in

* https://github.com/strogonoff/metanorma-csd/commit/0793f66d7643145dba33fca94217984e617ddfa9, https://github.com/strogonoff/metanorma-csd/commit/e14fc0b4eefe46c7c3cbeb6d4a7b105ad86ad2d5
* https://github.com/riboseinc/metanorma.com/commit/99d88cd63ebc61a4d0bd97db192197bd84c3d7b4, https://github.com/riboseinc/metanorma.com/commit/e0a989f791f6ef96bde1e42a3c7b7ad5fdb6360e

**Comment by ronaldtse** [comment]
@strogonoff thank you for the impressive work!! It looks awesome.

Layout wise:
* If I shrink my screen to some point the words touch my right screen border
* The left sidebar is often empty at the bottom when I scroll down, which is a bit awkward. It would be much better if we can provide a table of contents that follows the document and highlights the section I'm in?

**Comment by strogonoff** [comment]
@ronaldtse should be addressed by https://github.com/riboseinc/jekyll-theme-open-project/issues/43!

**Comment by ronaldtse** [comment]
Thanks!

**Comment by strogonoff** [comment]
The issues raised in https://github.com/riboseinc/metanorma.com/issues/25#issuecomment-427725310 were fixed in OP framework 1.1.17, more layout updates to come (centering the text & more adjustments)
